scripts/compress_jointcal_hsc_raw_test_data.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+"""Copy in the relevant raw files from RC/w_2020_14/DM-24359-sfm, zero out the
+imageHDUs in the raws.
+
+No need to re-compress with fpack, as they are all already CompImageHDUs.
+Requires that this be run on `lsst-dev`, and run from `testdata_jointcal/`, with
```

Comment:
To be clear, this has been run and those are the files that are part of this PR right?

Reply:
Yes.